neurom/check/structural_checks.py

```diff
@@ -27,111 +27,19 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Comment:
@lidakanari does your team use these checks? How important are they? The current plan is to drop them in version 2.

Reply:
this is resolved, right?

Reply:
Yes